package.json

```diff
@@ -28,15 +29,12 @@
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
-    "babel-cli": "^6.14.0",
-    "babel-eslint": "8.1.2",
-    "babel-loader": "^7.0.0",
-    "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "babel-plugin-transform-object-rest-spread": "^6.19.0",
-    "babel-plugin-transform-runtime": "^6.15.0",
```

Comment:
this was causing the bundle to be extra large -- regenerator runtime for async await